pkgs/top-level/python-packages.nix

```diff
@@ -3725,6 +3725,8 @@ self: super: with self; {
 
   fusepy = callPackage ../development/python-modules/fusepy { };
 
+  fusesoc = callPackage ../development/python-modules/fusesoc { };
```

Comment:
As this is an end-user program, not a library, it might be preferably exported from `pkgs/top-level/all-packages.nix`

Reply:
What about in `pkgs/top-level/aliases.nix`? That way people can use the python311 version if they need to for whatever reason.

Reply:
A user of the application shouldn't care what version of the language the application is using.  all-packages.nix is the right place for this.

[Relevant manual section](https://nixos.org/manual/nixpkgs/unstable/#contributing)

Reply:
im assuming i should remove it from here as well?

Reply:
> What about in `pkgs/top-level/aliases.nix`? That way people can use the python311 version if they need to for whatever reason.

New names should not be directly added to aliases